app/src/main/kotlin/com/hippo/ehviewer/ui/reader/PagerItem.kt

```diff
@@ -82,18 +86,41 @@ fun PagerItem(
                     }
                 }
             }
-            Image(
-                painter = painter,
-                contentDescription = null,
-                modifier = contentModifier.fillMaxSize(),
-                contentScale = contentScale,
-                colorFilter = when {
-                    grayScale && invert -> grayScaleAndInvertFilter
-                    grayScale -> grayScaleFilter
-                    invert -> invertFilter
-                    else -> null
-                },
-            )
+            if (image.hasQRCode) {
+                SubcomposeAsyncImage(
+                    model = ADSPlaceholderFile,
+                    contentDescription = null,
+                    modifier = contentModifier.fillMaxSize(),
```

Comment:
`modifier` cuz placeholder should not be zoomed.

Reply:
Done